numpy/distutils/ccompiler_opt.py

```diff
@@ -16,6 +16,14 @@
 import subprocess
 import textwrap
 
+# These flags are used to compile any C++ source within Numpy.
+# They are chosen to have very few runtime dependencies.
+NPY_CXX_FLAGS = [
+    '-std=c++11',  # Minimal standard version
+    '-D__STDC_VERSION__=0',  # for compatibility with C headers
+    '-fno-exceptions',  # no exception support
+    '-fno-rtti']  # no runtime type information
+
```

Comment:
I wonder if we should document these flags in [numpy/doc/source/user/building.rst](https://github.com/numpy/numpy/blob/main/doc/source/user/building.rst#supplying-additional-compiler-flags) as "the set of c++ features we support", xref #21299

Reply:
Yeah, that would make sense.